packages/smooth_app/lib/pages/product/common/product_list_page.dart

```diff
@@ -231,6 +231,60 @@ class _ProductListPageState extends State<ProductListPage>
         actionModeTitle: Text(appLocalizations.compare_products_appbar_title),
         actionModeSubTitle:
             Text(appLocalizations.compare_products_appbar_subtitle),
+        actionModeActions: <Widget>[
+          IconButton(
+            icon: const Icon(Icons.delete),
+            onPressed: () async {
+              if (_selectedBarcodes.isNotEmpty) {
+                await showDialog<bool>(
+                  context: context,
+                  builder: (BuildContext context) {
+                    return SmoothAlertDialog(
+                      body: Text(
+                        appLocalizations.confirm_clear_selected_user_list,
```

Comment:
I'm sorry for this change, but could you split the sentence in two?

First: 'You're about to clear selected items in your history.'
Second: 'Are you sure you want to continue?'

And between the two, you can add a few space (don't hesitate to use our constants).
It will be easier to read like so.